casper/src/test/scala/coop/rchain/casper/api/CreateBlockAPITest.scala

```diff
@@ -72,9 +82,9 @@ class CreateBlockAPITest extends FlatSpec with Matchers {
       result       <- testProgram(blockApiLock)(casperRef)
     } yield result).value.unsafeRunSync.right.get
 
-    response1.success shouldBe true
-    response2.success shouldBe false
-    response2.message shouldBe "Error: There is another propose in progress."
+    response1.isRight shouldBe true
```

Comment:
When this test fails you will get a rather vague `false was not true` message. A more informative error would come from `response1 shouldBe a[Right]`